File: dongyimai_manager_web/src/main/webapp/js/service/payLogService.js
app.service('payLogService',function ($http) {
    this.findAllPayLog=function(){
       return $http.get('../paylog/findAllPayLog.do');
    }

    //按分页查询
    this.findByPage=function (page,rows) {
        return $http.get('../paylog/findPage.do?page='+page+'&rows='+rows);
    }


})

File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/PayLogService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbPayLog;

import java.util.List;

public interface PayLogService {

    /**
     * 查询全部支付日志
     * @return
     */
    public List<TbPayLog> findAllPayLog();

    /**
     * 返回分页列表
     * @param pageNum
     * @param pageSize
     * @return
     */
    public PageResult findPage( int pageNum, int pageSize);
}
